src/BuildIt.Forms/BuildIt.Forms.Controls/CameraPreviewControl.xaml

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<Frame xmlns="http://xamarin.com/schemas/2014/forms"
+       xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+       x:Class="BuildIt.Forms.Controls.CameraPreviewControl"
+       HeightRequest="240"
+       WidthRequest="240"
+       BackgroundColor="Black"
```

Comment:
Should we make this transparent by default? Also, what happens if the user sets the height/width of the control, does it alter the size of the Frame?

Reply:
I think transparent by default is a bit strange, but I can do. In fact, just getting rid of the backgroundcolor setting will default to transparent.
The height/width can be overridden when adding the control in via XAML, these are just default values. The camera preview window should always scale to whatever the size of the control is in a "UniformToFill" manner (except in the case of iOS, where it will be scalled in a UniformToFit, and I'm not 100% sure why...)